packages/zoe/src/contracts/newSwap/collectFees.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+
+import { amountMath } from '@agoric/ertp';
+import { trade } from '../../contractSupport';
+
+export const makeMakeCollectFeesInvitation = (zcf, feeSeat, centralBrand) => {
+  const collectFees = seat => {
+    const allocation = feeSeat.getAmountAllocated('RUN', centralBrand);
+    trade(
+      zcf,
+      {
+        seat,
+        gains: { RUN: allocation },
+      },
+      {
+        seat: feeSeat,
+        gains: { RUN: amountMath.makeEmpty(centralBrand) },
+      },
+    );
+    seat.exit();
+    return `paid out ${allocation.value}`;
+  };
+
+  const makeCollectFeesInvitation = () =>
+    zcf.makeInvitation(collectFees, 'collect Fees');
```

Comment:
There's a naming convention for the strings here to better work in the wallet. Should this use that? (midcaps?)

Reply:
Where do I find the naming convention?  First I've heard of it.